buddyboss/buddyboss-platform
# Repeater Fields cant be saved

### All Submissions:






### Changes proposed in this Pull Request:



Fixes #1030 .

### How to test the changes in this Pull Request:
1. Create a Repeater Field
2. Add 3 single line field, 1 date and 1 radio button
3. Go to frontend, Profile > Edit
4. Go to Repeater Field and add values
4. Add again another set of repeater fields and save this
6. Delete the second repeater set created and hit save
7. Refresh the page and see its works or not

### Proof Screenshots or Video
http://somup.com/cY1DXVQrRc


### Other information:
